src/middleware.ts

```diff
@@ -0,0 +1,14 @@
+import { NextResponse, NextRequest } from 'next/server'
+
+export function middleware(request: NextRequest) {
+  let cookie = request.cookies.get('Langugae')?.value
+
+  let response = new NextResponse()
+  if (cookie) {
+    response.headers.set('Langugae', `${cookie}`)
```

Comment:
As Will pointed out, this is misspelled. In case we will use the cookie solution this should be fixed ofc :)